src/components/pages/StudentHome/StudentStoryCard.js

```diff
@@ -1,15 +1,65 @@
 import React, { useState } from 'react';
 import '../../../styles/StudentStyles/index.less';
 import { connect } from 'react-redux';
+import { Card, Button, Form } from 'antd';
+import { SendOutlined } from '@ant-design/icons';
 
 function StudentStoryCard(prop) {
   return (
-    <div className="student-story-card">
-      <img src="https://via.placeholder.com/50" alt="placeholder"></img>
-      <h3>Teacher Name</h3>
-      <h4>Coder Heroes</h4>
-      <h4>9 Minutes Ago</h4>
-    </div>
+    <Card className="student-story-card">
+      <div className="top-story-card">
+        <section className="top-left">
+          <img src="https://via.placeholder.com/50" alt="placeholder"></img>
+          <div>
+            <h2>Teacher Name</h2>
+          </div>
+        </section>
+        <section className="top-right">
+          <h4>9 Minutes Ago</h4>
+        </section>
+      </div>
+      <section className="teacher-message">
+        <h4>Teacher message</h4>
+        <a href="www.google.com">
+          www.google.com
+        </a>
+      </section>
+      <div className="story-photo">
+        <p></p>
+      </div>
+      <section className="student-comment">
```

Comment:
I like that you use sections in addition to divs (but not just divs), semantic html is the way to go. And naming divs helps accessibility as well!